yt_dlp/extractor/manoto.py

```diff
@@ -11,25 +11,50 @@ class ManotoTVIE(InfoExtractor):
         'url': 'https://www.manototv.com/episode/12576',
         'info_dict': {
             'id': '12576',
-            'title': 'Seh Mah Taatili',
-            'description': 'مجموعه ای از فیلم های سینمای کلاسیک ایران',
+            'series': 'فیلم های ایرانی',
+            'season_number': 0,
+            'episode_number': 0,
+            'episode_id': 'Seh Mah Taatili',
+            'duration': 5400,
+            'view_count': 10550,
+            'categories': ['سرگرمی'],
+            'title': 'سه ماه تعطیلی',
+            'description': 'سه ماه تعطیلی فیلمی به کارگردانی و نویسندگی شاپور قریب ساختهٔ سال ۱۳۵۶ است.<br/><br/>',
             'thumbnail': r're:^https?://.*\.jpeg$',
-            'ext': 'm3u8',
+            'ext': 'mp4'
         }
     }
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
         episode_json = self._download_json('https://dak1vd5vmi7x6.cloudfront.net/api/v1/publicrole/showmodule/episodedetails?id=' + video_id, video_id)
         details = episode_json.get('details', {})
-        title = details.get('analyticsEpisodeTitle')
+        series = details.get('showTitle')
+        season_number = details.get('analyticsSeasonNumber')
+        if not(season_number.isdigit()):
+            season_number = 0
+        episode_number = details.get('episodeNumber')
+        if not(episode_number.isdigit()):
+            episode_number = 0
```

Comment:
Why set it to 0?

Reply:
Sometimes these values are '' or some string of characters.

Reply:
```suggestion
        season_number = int_or_none(details.get('analyticsSeasonNumber'))
        episode_number = int_or_none(details.get('episodeNumber'))
```